indolaw-parser/parser_ui.py

```diff
@@ -24,8 +24,9 @@ def print_dashed_line():
     print('- - - - - - - -')
 
 
-def print_yes_no():
-    print(f"{colored('y', 'green')} / {colored('n', 'red')}")
+def print_yes_no_undo():
+    print(f"Press {colored('z', 'blue')} to undo")
```

Comment:
can we make 'u' undo? the 'z' choice is pretty clever (since cmd+z right?) but 'u' is closer to 'y' and 'n' on the keyboard and less awkward to type (since z is usually the responsibility of the left pinky)

Reply:
also if we change it to 'u' we can get rid of this line, and in the line below we can just make it '(u)ndo'